manifests/07-downloads-deployment.yaml

```diff
@@ -135,7 +135,7 @@ spec:
               for directory in directories:
                   write_index(
                       path=os.path.join(root, directory, 'index.html'),
-                      message='<p>Directory listings are disabled.  See <a href="">here</a> for available content.</p>'.format(root_link),
+                      message='<p>Directory listings are disabled.  See <a href="{}">here</a> for available content.</p>'.format(root_link),
```

Comment:
Unrelated, but thanks for this fix :)

Reply:
Found this testing the script with python3, but yes, it was broken before too.  The root index.html still doesn't work as intended though, nor did it before, but I'm not sure how to fix it yet.  (Nothing to do with this PR though.)

Reply:
@spadgett something to potentially clean up in the future

Reply:
#464 has the remaining cleanup